sos/policies/__init__.py

```diff
@@ -923,5 +955,242 @@ def pre_work(self):
 
         return
 
+    def prompt_for_upload_user(self):
+        """Should be overridden by policies to determine if a user needs to
+        be provided or not
+        """
+        if (not self.commons['cmdlineopts'].upload_user and not
```

Comment:
Isnt it better to test here `self.upload_user` instead of cmdlineopts param? Both shall have the same value, just everywhere else we test `self.upload_user` or `upload_password`.

Reply:
Yup, this was a straggler from when I moved everything to `self.upload_*` from the cmdline opts at the beginning of the upload logic. Fixing.